guides/Identities.md

```diff
@@ -0,0 +1,78 @@
+
+## Site visitor
+
+An anonymous site visitor who hasn't logged in. Methods that can be called by this identity usually involve operations specific to a particular visitor. These include accessing a list of products, creating and managing a cart, or accessing a login page to authenticate as a member.
+
+**Context**:  
+- [Headless](https://dev.wix.com/docs/go-headless/getting-started/about-headless/about-wix-headless) flows can make calls as site visitors.
+
+## Site member
+
+A site member who has logged in. Methods that can be called by this identity usually involve operations specific to a registered member. These include accessing or managing personal data.
+
+**Context**:   
+- [Headless](https://dev.wix.com/docs/go-headless/getting-started/about-headless/about-wix-headless) flows can make calls as site members.
+
+## Wix user
+
+A Wix user is someone who is logged into their account on **.wix.com**. Users can be site owners or site collaborators, as well as app owners and collaborators who create test sites. Users who create a site are automatically designated as the owner of that site. 
+
+Methods that can be called by this identity usually involve site maintenance operations. These include managing site products, media, and marketing campaigns.
+
+Site owners can invite collaborators and assign them specific [user roles](https://support.wix.com/en/article/roles-permissions-overview) to control permissions. In this case, the methods that the app can access when making calls as a collaborator are limited by the collaborator's user roles.
+
+**Context**:   
+- [Headless](https://dev.wix.com/docs/go-headless/getting-started/about-headless/about-wix-headless) flows can make calls as Wix Users.
+- Apps can make calls [on behalf of Wix users](https://dev.wix.com/docs/build-apps/develop-your-app/access/authentication/about-authentication#authentication-on-behalf-of-a-wix-user).
+
+## Wix app
+
+Wix apps are packages of reusable functionality that users can add to their sites. When an app is installed or updated on a Wix site, Wix generates an [app instance](https://dev.wix.com/docs/build-apps/develop-your-app/access/app-instances/about-app-instances) with a unique ID. This ID represents the installed version of the app on that particular site. 
+A call authenticated with the Wix app identity is a call made by an app for a specific app instance.
+
+Methods that can be called by this identity usually involve site-level operations. These include managing the site's data collections, contacts, or products.
+
+Each method requires certain permissions when called by a Wix App. To check which permissions a method requires, refer to its API reference.
+
+Learn more about [configuring app permissions](https://dev.wix.com/docs/build-apps/develop-your-app/access/authorization/configure-permissions-for-your-app).
+
+**Context**:
+- Apps can make calls as apps.
+
+## API key admin
```

Comment:
I recall discussing with Netanel (or maybe Gonen?) that API Keys aren’t technically considered identities, especially in the context of apps since API Keys aren't relevant there. Did you check with them about this section?

I'm wondering if we may want to avoid labeling API Keys as "identities" in the list, and just have a short section here explaining that API Keys grant specific permissions for admin operations and link to the API Key article: https://dev.wix.com/docs/rest/articles/getting-started/api-keys

Also, could a specific method state "can only be called by an API key"? This restriction is common for other identity types, but I’m not sure if it applies to API Keys.

Reply:
Apps aren't supposed to use API keys, but if they do, wix doesn't know that they are apps
all we know is that an api key was used - if we don't consider this an identity, then calls made with an api key have no identity, which I think is even more confusing